buildSrc/src/main/kotlin/com/datadog/gradle/config/FlavorBuildConfig.kt

```diff
@@ -23,6 +24,33 @@ fun sampleAppConfig(filePath: String): SampleAppConfig {
     }
 }
 
+@Suppress("UnstableApiUsage")
+fun ApplicationDefaultConfig.configureFlavorForBenchmark(
```

Comment:
do you plan to use this several times ? In case not maybe it should live in the `build.gradle` of the `benchmark` application

Reply:
I agree with @ambushwork , it makes sense to keep this here